custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -14,6 +14,25 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
                 $location.search('location_name', loc.name);
             }
         };
+        vm.filtersOpen = false;
```

Comment:
Probably could have broken this commit in two. But it's doing two things: 

1. Porting the mobile filter handling into the `BaseFilterController` class I made in the previous commit, which allows it to be shared between program summary and maps views.
2. Removing the redundant `moveToLocation` function from program summary, by having it extend the `BaseFilterController`

This section is in service of change 1.